sdk/documentintelligence/azure-ai-documentintelligence/azure/ai/documentintelligence/_patch.py

```diff
@@ -6,9 +6,86 @@
 
 Follow our quickstart for examples: https://aka.ms/azsdk/python/dpcodegen/python/customize
 """
-from typing import List
+from typing import Any, List, Union
 
-__all__: List[str] = []  # Add all objects you want publicly available to users at this package level
+from azure.core.credentials import AzureKeyCredential, TokenCredential
+
+from ._client import(
+    DocumentIntelligenceClient as DIClientGenerated,
+    DocumentIntelligenceAdministrationClient as DIAClientGenerated,
+)
+
+
+class DocumentIntelligenceClient(DIClientGenerated):  # pylint: disable=client-accepts-api-version-keyword
+    """DocumentIntelligenceClient.
+
+    :param endpoint: The Document Intelligence service endpoint. Required.
+    :type endpoint: str
+    :param credential: Credential needed for the client to connect to Azure. Is either a
+     AzureKeyCredential type or a TokenCredential type. Required.
+    :type credential: ~azure.core.credentials.AzureKeyCredential or
+     ~azure.core.credentials.TokenCredential
+    :keyword api_version: The API version to use for this operation. Default value is
+     "2023-10-31-preview". Note that overriding this default value may result in unsupported
+     behavior.
+    :paramtype api_version: str
+    :keyword int polling_interval: Default waiting time between two polls for LRO operations if no
```

Comment:
Since it's not in the method signature, remove it. We're doing this for autorest as well
https://github.com/Azure/autorest.python/issues/2310